### DISCUSSION TITLE
undo a command

### DISCUSSION THREAD

**Discussion by sylvainmouquet** [containers/buildah#5461] [Ideas]
Hello,

It's possible to undo a command with buildah ?

Thanks

**Comment by rhatdan** [comment]
Not really sure what you mean, but most likely no, you can add another layer to remove content added in other layers.
But in a lot of cases it would be better to recreate the image without the content in it in the first place.

**Comment by sylvainmouquet** [comment]
By example i have 4 layers, i would like to undo the layer 4 to restart after the layer 3 without running previous layers.

**Reply by rhatdan** [reply]
If you run the Containerifle without the 4th line that creates the image, it will use cached versions of the first three.

**Reply by sylvainmouquet** [reply]
yes but it needs to restart from 0. I think it should be better to be like a stack where we can pop and pull.
